sklearn/kernel_ridge/kernel_ridge.py

```diff
@@ -181,5 +179,12 @@ def predict(self, X):
             Returns predicted values.
         """
         check_is_fitted(self, ["X_fit_", "dual_coef_"])
+        # Input validation
         K = self._get_kernel(X, self.X_fit_)
-        return np.dot(K, self.dual_coef_)
+        y = np.dot(K, self.dual_coef_)
```

Comment:
I suspect KernelRidgeClassifier should also have a `decision_function` reporting this

Reply:
I don't understand very well your comment.
I just write that, in case of working as a regressor, that's it, but working as a classifier, a inverse transformation (from encoded labels to original labels, using `LabelBinarizer`)

Reply:
Multiclass (linear) classifiers often provide `decision_function` which reports a score for each class for each sample, such that the argmax is the predicted class.

Reply:
Ah, I see, for cross validation, right? I'll take a look about how it is implemented in other classifiers

Reply:
Ok, `decision_function` was easy to implement, because it is the first part of the prediction in multiclass problems. Then, as you pointed, something like argmax is applied, but through `LabelBinarizer`.